### PR TITLE
fixes taxi pods not being constructable

### LINKED ISSUE #15682: You can't actually build a taxi spacepod
(WEB REPORT BY: gutlesscoward REMOTE: 172.93.109.202:7777)
>Revision (Should be above if you're viewing this from ingame!)
eccd8396b3232b1871b5acef4d92ad83371ce94e 

>General description of the issue
if you try to build a taxi spacepod (doing the entire spacepod building process, but using taxi spacepod armor instead of civvie) a civilian spacepod comes out instead.

>What you expected to happen
get a taxi spacepod

>What actually happened
get a civilian spacepod

>Steps to reproduce if possible
1.build a spacepod. when at  the step where you put in civvie pod armor, put in taxi pod armor.

### PR DESCRIPTION
**Pull request by MadmanMartian**
Construction datums are a fickle fiend

closes #15682

Seems the mapcleaner does not take 'taxi station' into account? Might be an oversight of some form

Current problems
 - All those changed lines on taxistation (Apparently irrelevant as we don't use it anyway)

:cl: 
 * bugfix: You can now actually make taxi pods. Sorry folks

### CODE REVIEW

**Comment by N3X15** [comment]
This seems like a pretty big thing to have missed.  Have you tested to make sure it works this time?

**Comment by SonixApache** [comment]
we don't talk about taxi anymore

**Comment by MadmanMartian** [comment]
Oh, okay. Well I've changed its mapped-in pod armor just in case anyway

**Comment by MadmanMartian** [comment]
And that should be good to go. Tested it, was able to construct both pod types

**Comment by DamianX** [comment]
that diff is quite big

File: code/WorkInProgress/pomf/spacepods/construction.dm [hunk 1]
```diff
@@ -244,121 +235,56 @@
...
 				),
-				// 6. Core inserted
+				// 2. Loose armor
 				list(
-					Co_DESC = "A naked space pod with a loose core.",
+					Co_DESC = "A space pod with unsecured armor.",
 					Co_BACKSTEP = list(
 						Co_KEY      = /obj/item/weapon/crowbar,
-						Co_VIS_MSG  = "{USER} delicately remove{s} the core from {HOLDER} with a crowbar."
+						Co_VIS_MSG  = "{USER} prie{s} off {HOLDER}'s armor."
```
Comment (Exxion) [inline code/WorkInProgress/pomf/spacepods/construction.dm]: >`prie`

Comment (MadmanMartian) [reply code/WorkInProgress/pomf/spacepods/construction.dm]: That's just copypasted from the old pod construction code. Good spot

**Comment by MadmanMartian** [comment]
@DamianX it was because of taxistation not being covered by mapcleaner

**Comment by MadmanMartian** [comment]
The absolute madman

**Pull request merged**